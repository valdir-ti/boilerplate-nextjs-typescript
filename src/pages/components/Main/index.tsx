const Main = () => (
  <main>
    <h3>React Avan√ßado</h3>
  </main>
)

export default Main
