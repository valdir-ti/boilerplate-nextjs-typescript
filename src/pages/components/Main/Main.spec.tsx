import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'

import Main from '.'

describe('<Main />', () => {
  it('should render a heading', () => {
    render(<Main />)
    expect(
      screen.getByRole('heading', { name: /react avanÃ§ado/i })
    ).toBeInTheDocument()
  })

  it('should match a snapshot', () => {
    const { container } = render(<Main />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
