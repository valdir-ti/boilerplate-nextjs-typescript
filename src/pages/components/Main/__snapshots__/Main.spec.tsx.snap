// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should match a snapshot 1`] = `
<main>
  <h3>
    React Avan√ßado
  </h3>
</main>
`;
